### ISSUE TITLE
Have to add another footer

### ISSUE THREAD

**Issue by sifa123** [sifa123/Dev-Kitchen#33]
As we have a navbar in two sections, just like that we have to add another footer section.

![Web capture_11-7-2021_224610_127 0 0 1](https://user-images.githubusercontent.com/63462931/125305882-8c8ef500-e350-11eb-8dde-0fab3b921f1d.jpeg)

This footer is already added. You have to add another footer before that, which will contain About Dev Kitchen, Photo Gallery, Business Hours.
![20210710_103830](https://user-images.githubusercontent.com/63462931/125306262-df68ac80-e350-11eb-95e2-f2f9cd5030a3.jpg)
@harshita0802goswami would you like to work on that?

**Comment by heet-kumar** [comment]
@sifa123 above the footer section you want to add this pencil drawn layout in which page

**Comment by heet-kumar** [comment]
I think in all the pages

**Comment by heet-kumar** [comment]
@sifa123 as a suggestion footer should contain only About Dev-kitchen, Opening Hours, Location(Address), and Site Map (hyperlinks to the other pages) i think it dosen't contain gallery in footer . If you want to add small gallery section add above the footer like this : 
![Screenshot (765)](https://user-images.githubusercontent.com/78725162/125339971-83615080-e36f-11eb-9d85-2f8477ec974a.png)
it will look much better .

I apologies if you mind my suggestion .

**Comment by heet-kumar** [comment]
by the way I want to work on this issue so if possible then assign me.

**Comment by heet-kumar** [comment]
under the DCP'21

**Comment by sifa123** [comment]
@heet-kumar, yes, it should be on all pages. 

- I appreciate your suggestion. As you can see in the navbar, we have already a gallery page, so we will not add a gallery section to the home page. But as I have drawn on paper, this photo gallery will be small in size like this 

![Screenshot (168)](https://user-images.githubusercontent.com/63462931/125418000-625ef14d-ffbe-40f6-ac4a-b332c5d66968.png)

- You can also use our DevIncept photo as a background and on DevIncept photo add a like button which will take the user to Facebook page, add a follow button for linkedin and show total followers on LinkedIn.

**Comment by heet-kumar** [comment]
@sifa123  buttons are required they are already created na google, linkedin , Insta, etc all are already created na   
 ![image](https://user-images.githubusercontent.com/78725162/125443622-986fe6be-4e55-4a69-a739-7eff41b2c7b5.png)

**Comment by sifa123** [comment]
Yeah, it will look bad, then could you design this as I have said at first (About Dev Kitchen, photo gallery and opening closing hours)?

**Comment by heet-kumar** [comment]
ya sure

**Comment by heet-kumar** [comment]
for gallery photo can i use any photos or else you will provide it .

**Comment by sifa123** [comment]
Anything you want but base one restaurant :smiley:

**Comment by aditya687** [comment]
Can I try this issue @sifa123 ?

**Comment by heet-kumar** [comment]
@aditya687 i am working on this issue so find it other issue

**Comment by aditya687** [comment]
No problem, keep going on.
